fast64_internal/f3d/f3d_gbi.py

```diff
@@ -1673,6 +1680,24 @@ def getLightMWO_b(self, n):
             raise PluginError("Invalid G_MWO_b value for lights: " + n)
 
 
+g_F3D = {"GBI": None, "f3d_type": None, "isHWv1": None}
+
+
+def get_cached_F3D_GBI(f3d_type: str, isHWv1: bool) -> F3D:
+    """Get constructed/cached F3D class"""
+    global g_F3D
```

Comment:
```suggestion
```
only need that if you were to do `g_F3D = `, accessing properties or items will edit g_F3D anyway